gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -157,13 +158,37 @@ def _read_variants(pipeline, known_args):
   return variants
 
 
-def _use_beam_header_merger(known_args):
+def _beam_merge_headers(known_args, pipeline_args):
   if known_args.representative_header_file:
-    return False
+    return
+
   match_results = FileSystems.match([known_args.input_pattern])
```

Comment:
we should only do this if '--optimize_for_large_inputs' is false (it takes a while to get the list of files and would be unnecessary if the user has already hinted to use large inputs).

Reply:
Done.